homework-submission/react-hyf/react-hyf-client/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import Home from './Home';
+import PreviousHomeworks from './PreviousHomeworks';
+import HomeworkWeek3 from './HomeworkWeek3';
+import week1 from './Week1';
+
+class App extends Component {
+  render() {
+    return (
+      <BrowserRouter>
+        <div>
+          <Route exact path="/" component={Home}></Route>
```

Comment:
Cool that you made a menu for the different homework results

Reply:
thanks man!